src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -552,7 +538,7 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
         List<String> studentList = new ArrayList<>();
         if (!repositoryExportOptions.isExportAllStudents()) {
             studentIds = studentIds.replaceAll(" ", "");
-            studentList = Arrays.asList(studentIds.split("\\s*,\\s*"));
+            studentList = Arrays.asList(studentIds.split(","));
```

Comment:
Is this change really correct? I am note sure but I thought the previous version remove all whitespaces, right?

Reply:
Yes, but it is redundant since we already remove all whitespaces in the previous line.

Reply:
Only spaces. What happens to other whitespaces?

Reply:
I think we should change the previous line to 

`studentIds = studentIds.replaceAll("\\s+", "");`

Reply:
then it would be fine

Reply:
Fixed